audittail-helm-library/tests/consumerapp/templates/deployment.yaml

```diff
@@ -16,13 +16,22 @@ spec:
     spec:
       initContainers:
         {{- include "audittail.initContainer" .| nindent 8 }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
       containers:
-        - image: nging
+        - image: nginx:1.0
           name: nginx
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
           args:
             - --audit-log-path={{ template "audittail.auditLogPath" }}
           volumeMounts:
             {{- include "audittail.volumeMount" . | nindent 12}}
         {{- include "audittail.sidecarContainer" .| nindent 8 }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
```

Comment:
Could we make these the default and not rely on the user setting this? The idea is to promote good security practices.